Gopkg.toml

```diff
@@ -28,6 +28,14 @@ required = [
   name = "gopkg.in/yaml.v2"
   version = "v2.2.4"
 
+[[override]]
+  name = "github.com/google/gofuzz"
+  # This is the only a single commit ahead of apimachinery's
+  # version.
```

Comment:
include which version of apimachinery, so that it is clearer when its time to revisit?

Reply:
PR is outstanding - see https://github.com/knative/pkg/pull/1010